src/main.rs

```diff
@@ -369,7 +369,10 @@ async fn root_post(
         }
     });
 
-    debug!("started: {}", uuid);
+    info!(
+        "job started. job_id={uuid}, user_id={}",
+        user.read().await.uid
+    );
```

Comment:
Why not make this a structured log?

Reply:
We should, but last time I've tried it, it did not work, apparently due to the "structured" logging being unstable in the upstream crate, not sure, but all the structured fields were just silently dropped. Since time was critical here, I went with the simplest solution to avoid guesswork

Reply:
fine with me

Reply:
https://github.com/profianinc/benefice/issues/96